packages/core/src/tracing/sdk/sdk.js

```diff
@@ -151,14 +180,20 @@ module.exports = exports = function(isCallbackApi) {
     completeSpan(error, span, tags);
   }
 
+  /**
+   * @param {string} name
+   * @param {string} tags
+   * @param {Function | string} callback
+   * @returns
```

Comment:
*question:* What does an empty `@returns` annotation do? There are a couple of them in this PR.